app/models/concerns/arel_helper.rb

```diff
@@ -82,6 +82,15 @@ def any(*args)
       nf 'ANY', args
     end
 
+    def array_agg(*args)
+      nf 'ARRAY_AGG', args
+    end
+
+    def array(*args)
```

Comment:
I wonder if this should be `sql_array` or something.  Seems like mixing in an `array` function wherever we include ArelHelper is a bit dangerous.